### PR TITLE
Ports TV-camera assembly from Bay

### PR DESCRIPTION
**Pull request by Runa-Dacino**
Copies https://github.com/Baystation12/Baystation12/pull/14875

I've recently noticed downstream you can't build TV Camera drones on demand.

How to build drones:

1. Get Robot head
2. Infrared sensor -> Makes the assembly. Looks like robot head.
3. Add robot camera
4. Add Tape recorder
5. Add 3 wires
6. Use wirecutters on assembly
7. Add 1 steel
8. Done

### CODE REVIEW

File: code/game/objects/items/devices/tvcamera.dm [hunk 1]
```diff
@@ -93,3 +93,78 @@
...
+//Using camcorder icon as I can't sprite. 
+//Using robohead because of restricting to roboticist		
+/obj/item/weapon/TVAssembly
+	name = "TV Camera assembly"
+	desc = "A robotic head with an infrared sensor inside"
+	icon = 'icons/obj/robot_parts.dmi'
+	icon_state = "head"
+	item_state = "head"
+	var/buildstep = 0
+	w_class = ITEM_SIZE_LARGE
```
Comment (Meghan-Rossi) [inline code/game/objects/items/devices/tvcamera.dm:119]: ITEMSIZE_LARGE

File: code/game/objects/items/devices/tvcamera.dm [hunk 2]
```diff
@@ -93,3 +93,78 @@
...
+					to_chat(user, "<span class='notice'>You need three cable coils to wire the devices.</span>")
+					..()
+					return
+				C.use(3)
+				buildstep++
+				to_chat(user, "<span class='notice'>You wire the assembly</span>")
+				desc = "This TV camera assembly has wires sticking out"
+				return
+		if(3)
+			if(istype(W, /obj/item/weapon/wirecutters))
```
Comment (Meghan-Rossi) [inline code/game/objects/items/devices/tvcamera.dm:153]: It's /obj/item/weapon/tool/wirecutters, except that that doesn't account for jaws of life.  if(W.is_wirecutter()) would account for that.

**Comment by Shadow-Quill** [comment]
Still needs to be updated

File: code/game/objects/items/devices/tvcamera.dm [hunk 3]
```diff
@@ -93,3 +93,78 @@
 		H.update_inv_l_hand()
 		H.update_inv_belt()
 
+//Assembly by roboticist
+	
+		
+/obj/item/robot_parts/head/attackby(var/obj/item/device/assembly/S, mob/user as mob)
+	if ((!istype(S, /obj/item/device/assembly/infra)))
```
Comment (Shadow-Quill) [inline code/game/objects/items/devices/tvcamera.dm:100]: Extraneous parentheses here

**Review by Shadow-Quill — CHANGES_REQUESTED** [review]

**Comment by Runa-Dacino** [comment]
Kinda had a sudden splurt of wanting to port this. Forgot about it. Will look at it later, sorry for pointless need for reviews >.>

**Pull request closed without merging**